src/MusicStore.Standalone/Program.cs

```diff
@@ -44,9 +44,10 @@ public static void Main(string[] args)
                 builder.UseKestrel();
             }
 
-            var host = builder.Build();
-
-            host.Run();
+            using (var host = builder.Build())
```

Comment:
Like @Eilon said after `host.Run()` returns the app immediately exits which will automatically dispose the host. I prefer the current way we have it since it looks cleaner.